### ISSUE TITLE
Decode error in vcard.py line 139 with python 3.x

### ISSUE THREAD

**Issue by croessner** [eventable/vobject#62]
Hi,

line 139 in vard.py needs changes.

```
133         if line.encoded:
134             if 'BASE64' in line.singletonparams:
135                 line.singletonparams.remove('BASE64')
136                 line.encoding_param = cls.base64string
137             encoding = getattr(line, 'encoding_param', None)
138             if encoding:

139                 line.value = base64.b64decode(line.value)

140             else:
141                 line.value = stringToTextValues(line.value)[0]
142             line.encoded=False

```
I do not know, if this is ok for python 2.7 as well. So I did not make a PR here.

The error is that "str" dows not have an attribute "decode"

**Comment by croessner** [comment]
Ah and of course you need to import base64

**Comment by olenchenko** [comment]
Hi.
I'm getting the same error:

`Traceback (most recent call last):
  File "/usr/local/lib/python3.6/wsgiref/handlers.py", line 137, in run
    self.result = application(self.environ, self.start_response)
  File "/usr/local/lib/python3.6/site-packages/radicale/__init__.py", line 419, in __call__
    environ, base_prefix, path, user)
  File "/usr/local/lib/python3.6/site-packages/radicale/__init__.py", line 680, in do_PUT
    items = list(vobject.readComponents(content or ""))
  File "/usr/local/lib/python3.6/site-packages/vobject/base.py", line 1109, in readComponents
    component.setBehaviorFromVersionLine(versionLine)
  File "/usr/local/lib/python3.6/site-packages/vobject/base.py", line 658, in setBehaviorFromVersionLine
    self.setBehavior(v)
  File "/usr/local/lib/python3.6/site-packages/vobject/base.py", line 170, in setBehavior
    obj.autoBehavior(True)
  File "/usr/local/lib/python3.6/site-packages/vobject/base.py", line 156, in autoBehavior
    self.behavior.decode(self)
  File "/usr/local/lib/python3.6/site-packages/vobject/vcard.py", line 143, in decode
    line.value = codecs.decode(line.value.encode("utf-8"), "base64")
binascii.Error: decoding with 'base64' codec failed (Error: Incorrect padding)`

but in line 143. I am using `vobject (0.9.4.1)`

Any suggestions?
Thanks in advance.

**Comment by olenchenko** [comment]
Just for the info - my problem was not related to vobject and I have fixed it.
So please don't mind my previous comment.

Thanks.

**Comment by wpercy** [comment]
Issue resolved in #49
